client/dashboard/profile-wizard/steps/plugins.js

```diff
@@ -181,18 +95,17 @@ class Plugins extends Component {
 					/>
 
 					<div className="woocommerce-profile-wizard__plugins-actions">
-						{ isError && (
+						{ hasErrors && (
 							<Button isPrimary onClick={ () => location.reload() }>
 								{ __( 'Retry', 'woocommerce-admin' ) }
 							</Button>
 						) }
 
-						{ ! isError &&
-							'activate' === step && (
-								<Button isPrimary isBusy={ isPending } onClick={ this.activatePlugins }>
-									{ __( 'Activate & continue', 'woocommerce-admin' ) }
-								</Button>
-							) }
+						{ ! ( hasErrors && 'activate' === step ) && (
+							<Button isPrimary isBusy={ isRequesting } onClick={ this.activatePlugins }>
```

Comment:
I see the buttons in "busy" state 👍 but the cursor is still showing clickable, which I did not try. Some CSS to change the cursor would be great.